### PR TITLE
os init_state: add new state OSINIT_OSIDLELOOP

### PR DESCRIPTION
**Pull request by GUIDINGLI**
## Summary

os init_state: add new state OSINIT_OSIDLELOOP

This is for distinguish we are in init idle or idle loop.
Use for assertion for sem_trywait & sem_wait

## Impact

wait/try wait in IDLE thread

## Testing

VELA

### CODE REVIEW

File: drivers/power/pm_register.c [hunk 1]
```diff
@@ -62,19 +62,11 @@ int pm_register(FAR struct pm_callback_s *callbacks)
 
   /* Add the new entry to the end of the list of registered callbacks */
 
-  if (OSINIT_OS_READY())
```
Comment (pkarashchenko) [inline drivers/power/pm_register.c:65]: Why this condition is removed? I mean why not keeping it and changing to `if (!OSINIT_OS_INITIALIZING())`?

Comment (GUIDINGLI) [reply drivers/power/pm_register.c:65]: Don't need.
Because when in system init state, we allow user to get lock.

**Review by xiaoxiang781216 — APPROVED** [review]

File: include/nuttx/init.h [hunk 2]
```diff
@@ -51,22 +52,23 @@
...
+  OSINIT_HARDWARE   = 4, /* MCU-specific hardware is initialized.  Hardware
                           * resources such as timers and device drivers are
                           * now available.  Low-level OS services sufficient
                           * to support the hardware are also available but
                           * the OS has not yet completed its full
                           * initialization. */
-  OSINIT_OSREADY   = 5   /* The OS is fully initialized and multi-tasking is
+  OSINIT_OSREADY    = 5, /* The OS is fully initialized and multi-tasking is
                           * active. */
+  OSINIT_OSIDLELOOP = 6  /* The OS enter idle loop */
```
Comment (xiaoxiang781216) [inline include/nuttx/init.h:71]: remove OS prefix

Comment (pkarashchenko) [reply include/nuttx/init.h:71]:
```suggestion
  OSINIT_IDLELOOP = 6  /* The OS enter idle loop */
```
?

**Review by xiaoxiang781216 — APPROVED** [review]

**Comment by masayuki2009** [comment]
```
====================================================================================
Configuration/Tool: sim/ostest
------------------------------------------------------------------------------------
  Cleaning...
  Configuring...
  Building NuttX...
In file included from /github/workspace/sources/nuttx/include/stdbool.h:54,
                 from semaphore/sem_wait.c:27:
semaphore/sem_wait.c: In function 'nxsem_wait':
Error: semaphore/sem_wait.c:80:15: error: implicit declaration of function 'OSINIT_OS_IDLELOOP'; did you mean 'OSINIT_IDLELOOP'? [-Werror=implicit-function-declaration]
   80 |   DEBUGASSERT(OSINIT_OS_IDLELOOP() && !sched_idletask());
      |               ^~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
make[1]: *** [Makefile:57: sem_wait.o] Error 1
In file included from /github/workspace/sources/nuttx/include/stdbool.h:54,
                 from semaphore/sem_trywait.c:27:
semaphore/sem_trywait.c: In function 'nxsem_trywait':
Error: semaphore/sem_trywait.c:76:15: error: implicit declaration of function 'OSINIT_OS_IDLELOOP'; did you mean 'OSINIT_IDLELOOP'? [-Werror=implicit-function-declaration]
   76 |   DEBUGASSERT(OSINIT_OS_IDLELOOP() && !sched_idletask());
      |               ^~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
make[1]: *** [Makefile:57: sem_trywait.o] Error 1
make[1]: Target 'libsched.a' not remade because of errors.
make: *** [tools/LibTargets.mk:59: sched/libsched.a] Error 2
make: Target 'all' not remade because of errors.
```

**Review by xiaoxiang781216 — APPROVED** [review]

**Review by pkarashchenko — APPROVED** [review]

**Comment by masayuki2009** [comment]
Hmm, spresense:elf now failed with this PR.

```
AEG
up_assert: Assertion failed at file:semaphore/sem_wait.c line: 80 task: Idle Task
irq_unexpected_isr: ERROR irq: 11
up_assert: Assertion failed at file:irq/irq_unexpectedisr.c line: 51 task: Idle Task
arm_registerdump: R0: 00000000 R1: 2d026f64 R2: 00000080  R3: 2d027850
arm_registerdump: R4: 2d026ee8 R5: 2d026ee8 R6: 00157b8c  FP: 2d028268
arm_registerdump: R8: 2d027850 SB: 2d027038 SL: 4ca6a801 R11: 3460eb23
arm_registerdump: IP: 00000000 SP: 2d028268 LR: 0d007673  PC: 0d007b34
arm_registerdump: xPSR: 61000000 BASEPRI: 00000080 CONTROL: 00000000
arm_registerdump: EXC_RETURN: ffffffe9
arm_dump_stack: User Stack:
arm_dump_stack: sp:     2d028138
arm_dump_stack:   base: 00000000
arm_dump_stack:   size: 00000000
arm_dump_stack: ERROR: User Stack pointer is not within the stack
arm_showtasks:    PID    PRI      USED     STACK   FILLED    COMMAND
```

**Comment by pkarashchenko** [comment]
@masayuki2009 do you think we should revert the change until we clarify your case?

**Comment by masayuki2009** [comment]
> @masayuki2009 do you think we should revert the change until we clarify your case?

@pkarashchenko 
I think so.

@GUIDINGLI 
Actually, maix-bit:kostest (with QEMU) also failed.

```
ABCDup_assert: Assertion failed at file:semaphore/sem_wait.c line: 80 task: Idle Task
irq_unexpected_isr: ERROR irq: 11
up_assert: Assertion failed at file:irq/irq_unexpectedisr.c line: 51 task: Idle Task
riscv_registerdump: EPC: 000000008000637e
riscv_registerdump: A0: 0000000000000000 A1: 0000000000000000 A2: 0000000000000000 A3: 0000000000000000
riscv_registerdump: A4: 0000000000000000 A5: 0000000000000000 A6: 0000000000000000 A7: 0000000000000000
riscv_registerdump: T0: 0000000000000000 T1: 0000000000000000 T2: 0000000000000000 T3: 0000000000000000
riscv_registerdump: T4: 0000000000000000 T5: 0000000000000000 T6: 0000000000000000
riscv_registerdump: S0: 0000000000000000 S1: 0000000000000000 S2: 0000000000000000 S3: 0000000000000000
riscv_registerdump: S4: 0000000000000000 S5: 0000000000000000 S6: 0000000000000000 S7: 0000000000000000
riscv_registerdump: S8: 0000000000000000 S9: 0000000000000000 S10: 0000000000000000 S11: 0000000000000000
riscv_registerdump: SP: 0000000080401990 FP: 0000000000000000 TP: 0000000000000000 RA: 0000000080006052
riscv_dumpstate: sp:     0000000080400800
```

**Comment by GUIDINGLI** [comment]
> 

Can you share the run & test cmd ?

**Comment by GUIDINGLI** [comment]
@masayuki2009
Please share the detail step how to reproduce, thanks

**Comment by masayuki2009** [comment]
@GUIDINGLI 

Please see nuttx/boards/risc-v/k210/maix-bit/README-qemu.txt  to run maix-bit:kostest.

**Comment by GUIDINGLI** [comment]
@masayuki2009 
I have run on configure of maix-bit:nsh, and ostest also execute successful.

ligd@Opt:~/platform/mainline/nuttx$ qemu-system-riscv64 -nographic -machine sifive_u -bios ./nuttx
```

NuttShell (NSH) NuttX-10.2.0
nsh> 
nsh> 
nsh> help
help usage:  help [-v] [<cmd>]

  .         cat       dd        false     ls        ps        test      unset     
  [         cd        df        free      mkrd      pwd       time      usleep    
  ?         cp        echo      help      mount     set       true      xd        
  basename  cmp       exec      hexdump   mv        sleep     truncate  
  break     dirname   exit      kill      printf    source    uname     

Builtin Apps:
  ostest    sh        getprime  hello     nsh       
nsh> 
nsh> ostest
stdio_test: write fd=1
stdio_test: Standard I/O Check: printf
stdio_test: write fd=2
stdio_test: Standard I/O Check: fprintf to stderr
ostest_main: putenv(Variable1=BadValue3)
ostest_main: setenv(Variable1, GoodValue1, TRUE)
ostest_main: setenv(Variable2, BadValue1, FALSE)
...
barrier_test: Thread 5 completed with result=0
barrier_test: Thread 6 completed with result=0
barrier_test: Thread 7 completed with result=0

End of test memory usage:
VARIABLE  BEFORE   AFTER
======== ======== ========
arena      1fd760   1fd760
ordblks         7        7
mxordblk   1f55d0   1f1df0
uordblks     7570     b3f0
fordblks   1f61f0   1f2370

Final memory usage:
VARIABLE  BEFORE   AFTER
======== ======== ========
arena      1fd760   1fd760
ordblks         1        7
mxordblk   1f7c50   1f1df0
uordblks     5b10     b3f0
fordblks   1f7c50   1f2370
user_main: Exiting
ostest_main: Exiting with status 0
nsh> 

```

But, with maix-bit:kostest, I meet this error:

```

ligd@Opt:~/platform/mainline/nuttx$ qemu-system-riscv64 -nographic -machine sifive_u -bios ./nuttx
ABCup_assert: Assertion failed at file:chip/k210_userspace.c line: 60 task: riscv_fault: EPC: 0000000080005382
riscv_fault: Fault IRQ=5
riscv_fault: A0: 0000000000000218 A1: 0000000000000217 A2: 0000000080401a78 A3: 000000000000002b
riscv_fault: A4: 0000000000000002 A5: 0000000000000218 A6: 000000000000006a A7: 000000000000006c
riscv_fault: T0: 000000000000002d T1: 0000000000000068 T2: 0000000000000030 T3: 0000000000000009
riscv_fault: T4: 000000000000002a T5: 000000000000002e T6: 00000000000001ff
riscv_fault: S0: 0000000000000000 S1: 0000000000000218 S2: 00000000804019f0 S3: 0000000000000049
riscv_fault: S4: 0000000000000000 S5: 0000000000000000 S6: 0000000080010da9 S7: 0000000000000a00
riscv_fault: S8: 0000000000001000 S9: 0000000000000800 S10: 0000000080401a80 S11: 0000000000000023
riscv_fault: SP: 0000000080401930 FP: 0000000000000000 TP: 0000000000000000 RA: 0000000080004ad2
up_assert: Assertion failed at file:chip/k210_irq_dispatch.c line: 91 task: riscv_fault: EPC: 0000000080005382
riscv_fault: Fault IRQ=5
riscv_fault: A0: 0000000000000218 A1: 0000000000000217 A2: 0000000080400818 A3: 000000000000002b
riscv_fault: A4: 0000000000000002 A5: 0000000000000218 A6: 000000000000006a A7: 000000000000006c
riscv_fault: T0: 000000000000002d T1: 0000000000000068 T2: 0000000000000030 T3: 00000000804000a0
riscv_fault: T4: 000000000000000a T5: 000000008000f3f6 T6: 00000000000001ff
riscv_fault: S0: 0000000000000000 S1: 0000000000000218 S2: 0000000080400790 S3: 000000000000004c
riscv_fault: S4: 0000000000000000 S5: 0000000000000000 S6: 0000000080010da9 S7: 0000000000000a00
riscv_fault: S8: 0000000000001000 S9: 0000000000000800 S10: 0000000080400820 S11: fffffffffffffffc
riscv_fault: SP: 00000000804006d0 FP: 0000000000000000 TP: 0000000000000000 RA: 0000000080004ad2
up_assert: Assertion failed at file:chip/k210_irq_dispatch.c line: 91 task: riscv_fault: EPC: 0000000080005382
riscv_fault: Fault IRQ=5
riscv_fault: A0: 0000000000000218 A1: 0000000000000217 A2: 0000000080400818 A3: 000000000000002b
riscv_fault: A4: 0000000000000002 A5: 0000000000000218 A6: 000000000000006a A7: 000000000000006c
riscv_fault: T0: 000000000000002d T1: 0000000000000068 T2: 0000000000000030 T3: 00000000804000a0
riscv_fault: T4: 000000000000000a T5: 000000008000f3f6 T6: 00000000000001ff

```

**Comment by masayuki2009** [comment]
>But, with maix-bit:kostest, I meet this error:

@GUIDINGLI 

As I wrote in README-qemu.txt

```
  NOTE: To run nuttx for kostest, gdb needs to be used to load both nuttx_user.elf and nuttx                                                                        
                                                                                                                                                                    
  $ qemu-system-riscv64 -nographic -machine sifive_u -s -S                                                                                                          
  $ riscv64-unknown-elf-gdb -ex 'target extended-remot:1234' -ex 'load nuttx_user.elf' -ex 'load nuttx' -ex 'c'        
```

**Comment by GUIDINGLI** [comment]
@masayuki2009 
From your testcase I found the error, many thanks.

Here is the new PR:
https://github.com/apache/incubator-nuttx/pull/5590

**Comment by GUIDINGLI** [comment]
@masayuki2009 
With my fix, there is still some other problem on maix-bit:kostest.
Is there something wrong on my environment ?

```

ligd@Opt:~/platform/mainline/nuttx$ riscv64-unknown-elf-gdb -ex 'target extended-remot:1234' -ex 'load nuttx_user.elf' -ex 'load nuttx'
GNU gdb (SiFive GDB 8.3.0-2019.08.0) 8.3
Copyright (C) 2019 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Type "show copying" and "show warranty" for details.
This GDB was configured as "--host=x86_64-linux-gnu --target=riscv64-unknown-elf".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<https://github.com/sifive/freedom-tools/issues>.
Find the GDB manual and other documentation resources online at:
    <http://www.gnu.org/software/gdb/documentation/>.

For help, type "help".
Type "apropos word" to search for commands related to "word".
Remote debugging using :1234
warning: No executable has been specified and target does not support
determining executable automatically.  Try using the "file" command.
0x0000000000001004 in ?? ()
Loading section .userspace, size 0x60 lma 0x80100000
Loading section .text, size 0xf3b8 lma 0x80100060
Loading section .data, size 0x58 lma 0x8010f418
Start address 0x801005de, load size 62576
Transfer rate: 6110 KB/sec, 1896 bytes/write.
Loading section .text, size 0x12500 lma 0x80000000
Loading section .data, size 0xac lma 0x80012500
Start address 0x80000000, load size 75180
Transfer rate: 6118 KB/sec, 1978 bytes/write.
(gdb) c
Continuing.

ligd@Opt:~/platform/mainline/nuttx$ qemu-system-riscv64 -nographic -machine sifive_u -s -S 
ABCDriscv_fault: EPC: 00000000800001ae
riscv_fault: Fault IRQ=2
riscv_fault: A0: 00000000801005de A1: 0000000000000001 A2: 0000000080401fa0 A3: 0000000080401fa0
riscv_fault: A4: 00000000801005de A5: 0000000000000001 A6: 0000000000000101 A7: 0000000000000000
riscv_fault: T0: 0000000000000000 T1: 0000000000000000 T2: 0000000000000000 T3: 0000000000000000
riscv_fault: T4: 0000000000000000 T5: 0000000000000000 T6: 0000000000000000
riscv_fault: S0: 0000000000000000 S1: 0000000000000000 S2: 0000000000000000 S3: 0000000000000000
riscv_fault: S4: 0000000000000000 S5: 0000000000000000 S6: 0000000000000000 S7: 0000000000000000
riscv_fault: S8: 0000000000000000 S9: 0000000000000000 S10: 0000000000000000 S11: 0000000000000000
riscv_fault: SP: 0000000080402b80 FP: 0000000000000000 TP: 0000000000000000 RA: 0000000080001a84
up_assert: Assertion failed at file:chip/k210_irq_dispatch.c line: 91 task: ostest_main
riscv_registerdump: EPC: 00000000800001ae
riscv_registerdump: A0: 0000000000000000 A1: 0000000000000000 A2: 0000000000000000 A3: 0000000000000000
riscv_registerdump: A4: 0000000000000000 A5: 0000000000000000 A6: 0000000000000000 A7: 0000000000000000
riscv_registerdump: T0: 0000000000000000 T1: 0000000000000000 T2: 0000000000000000 T3: 0000000000000000
riscv_registerdump: T4: 0000000000000000 T5: 0000000000000000 T6: 0000000000000000
riscv_registerdump: S0: 0000000000000000 S1: 0000000000000000 S2: 0000000000000000 S3: 0000000000000000
riscv_registerdump: S4: 0000000000000000 S5: 0000000000000000 S6: 0000000000000000 S7: 0000000000000000
riscv_registerdump: S8: 0000000000000000 S9: 0000000000000000 S10: 0000000000000000 S11: 0000000000000000
riscv_registerdump: SP: 0000000080402b80 FP: 0000000000000000 TP: 0000000000000000 RA: 0000000080001a84
riscv_dumpstate: sp:     0000000080400830
riscv_dumpstate: IRQ stack:
riscv_dumpstate:   base: 00000000804000b0
riscv_dumpstate:   size: 0000000000000800
riscv_stackdump: 0000000080400820: 00000be0 00000000 80006334 00000000 0000005b 00000000 8000fec8 00000000
riscv_stackdump: 0000000080400840: 00000000 00000000 00000002 00000000 80402a78 00000000 80400f60 00000000
riscv_stackdump: 0000000080400860: 00000002 00000000 80004782 00000000 00000002 00000000 80000260 00000000
riscv_stackdump: 0000000080400880: deadbeef deadbeef 00000000 00000000 00000000 00000000 00000000 00000000
riscv_dumpstate: sp:     0000000080402b80
riscv_dumpstate: User stack:
riscv_dumpstate:   base: 0000000080401fc0
riscv_dumpstate:   size: 0000000000000be0
riscv_dumpstate: ERROR: Stack pointer is not within allocated stack
riscv_stackdump: 0000000080401fc0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080401fe0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402000: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402020: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402040: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402060: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402080: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804020a0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804020c0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804020e0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402100: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402120: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402140: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402160: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402180: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804021a0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804021c0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804021e0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402200: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402220: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402240: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402260: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402280: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804022a0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804022c0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804022e0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402300: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402320: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402340: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402360: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402380: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804023a0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804023c0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804023e0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402400: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402420: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402440: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402460: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402480: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804024a0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804024c0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804024e0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402500: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402520: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402540: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402560: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402580: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804025a0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804025c0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804025e0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402600: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402620: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402640: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402660: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402680: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804026a0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804026c0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804026e0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402700: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402720: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402740: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402760: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402780: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804027a0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804027c0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804027e0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402800: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402820: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402840: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402860: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402880: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804028a0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804028c0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804028e0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402900: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402920: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402940: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402960: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402980: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804029a0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804029c0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 00000000804029e0: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402a00: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402a20: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402a40: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef
riscv_stackdump: 0000000080402a60: deadbeef deadbeef deadbeef deadbeef deadbeef deadbeef 800001ae 00000000
riscv_stackdump: 0000000080402a80: 80001a84 00000000 80402b80 00000000 deadbeef deadbeef 00000000 00000000
riscv_stackdump: 0000000080402aa0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
riscv_stackdump: 0000000080402ac0: 00000000 00000000 801005de 00000000 00000001 00000000 80401fa0 00000000
riscv_stackdump: 0000000080402ae0: 80401fa0 00000000 801005de 00000000 00000001 00000000 00000101 00000000
riscv_stackdump: 0000000080402b00: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
riscv_stackdump: 0000000080402b20: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
riscv_stackdump: 0000000080402b40: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
riscv_stackdump: 0000000080402b60: 00000000 00000000 00000000 00000000 00000000 00000000 00001800 0000000a
riscv_stackdump: 0000000080402b80: deadbeef deadbeef 80001a84 00000000 deadbeef deadbeef 00000000 00000000
riscv_showtasks:    PID    PRI      USED     STACK   FILLED    COMMAND
riscv_showtasks:   ----   ----       656      2048    32.0%    irq
riscv_dump_task:      0      0       608      2000    30.4%    Idle Task
riscv_dump_task:      1    100       296      3040     9.7%    ostest_main



```

**Pull request merged**